rorapi/v2/serializers.py

```diff
@@ -87,3 +88,15 @@ class MatchedOrganizationSerializer(serializers.Serializer):
 class MatchingResultSerializer(serializers.Serializer):
     number_of_results = serializers.IntegerField()
     items = MatchedOrganizationSerializer(many=True)
+
+
+class ClientSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Client
+        fields = ['email', 'name', 'institution_name', 'institution_ror', 'country_code', 'ror_use']
+
+    def validate_email(self, value):
+        """Validate the email format and ensure it's unique."""
+        if Client.objects.filter(email=value).exists():
+            raise serializers.ValidationError("A client with this email already exists.")
```

Comment:
Again, per the spec, email does not need to be unique - users should be able to register multiple client IDs with the same email. Validation should check that email looks like a valid email (@ symbol and domain).